src/theme/components/menu.ts

```diff
@@ -0,0 +1,26 @@
+import { menuAnatomy } from '@chakra-ui/anatomy'
+import { createMultiStyleConfigHelpers, defineStyle } from '@chakra-ui/react'
+
+const { definePartsStyle, defineMultiStyleConfig } = createMultiStyleConfigHelpers(menuAnatomy.keys)
+
+const baseStyle = definePartsStyle({
+  button: {
+    border: '3px solid red',
+  },
+  list: {
+    p: 0,
+    borderRadius: 'lg',
+    overflow: 'hidden',
+  },
+  item: {
+    px: 4,
+    py: 2.5,
+    display: 'flex',
+    gap: 2,
```

Comment:
Anything related to spacing should be defined as part of the `sizes` definition.